webrender/src/platform/macos/font.rs

```diff
@@ -454,8 +480,15 @@ impl FontContext {
         };
 
         let bitmap = is_bitmap_font(&ct_font);
-        let shape = font.transform.pre_scale(y_scale.recip() as f32, y_scale.recip() as f32);
-        let transform = if bitmap || shape.is_identity() {
+        let (mut shape, (x_offset, y_offset)) = if bitmap {
+            (FontTransform::identity(), (0.0, 0.0))
+        } else {
+            (font.transform.invert_scale(y_scale), font.get_subpx_offset(key))
+        };
+        if font.flags.contains(FontInstanceFlags::SYNTHETIC_ITALICS) {
```

Comment:
Same here as comment above.

Reply:
Can't do that here due to the fact that the initial transform is not necessarily the identity depending whether or not it is a bitmap. And on top of that, the synthetic italics can apply to possibly either case. So the is_identity() check is necessary here.